extensions/amp-loader/0.1/amp-loader.js

```diff
@@ -292,28 +218,34 @@ class LoaderBuilder {
   /**
    * Returns the custom logo for the element if there is one.
    * @private
-   * @return {?Element}
+   * @return {{
+   *  content: (!Element|undefined),
+   *  color: (string|undefined),
+   * }}
    */
   getCustomLogo_() {
     // Keeping the video logo here short term.
     // This is because there is no single CSS for all players, there is
     // video-interface but not all players implement it. Also the SVG is not
-    // that big. We may still want to move this out eventually.
+    // that big.
+    // We need to move most of loaders code out of v0 anyway, see
+    // https://github.com/ampproject/amphtml/issues/23108.
     if (isIframeVideoPlayerComponent(this.element_.tagName)) {
-      const svg = svgFor(this.element_);
-      const color = DEFAULT_LOGO_SPINNER_COLOR;
-      const svgNode = svg`
-        <path
-          class="i-amphtml-new-loader-white-on-shim"
-          d="M65,58.5V55c0-0.5-0.4-1-1-1H51c-0.5,0-1,0.5-1,1v10c0,0.6,0.5,1,1,1h13c0.6,0,1-0.4,1-1v-3.5l5,4v-11L65,58.5z"
-        ></path>
+      const html = htmlFor(this.element_);
+      const content = html`
+        <svg viewBox="0 0 72 72">
+          <path
+            class="i-amphtml-new-loader-white-on-shim"
+            fill="currentColor"
+            d="M41,34.5V31c0-0.5-0.4-1-1-1H27c-0.5,0-1,0.5-1,1v10c0,0.6,0.5,1,1,1h13c0.6,0,1-0.4,1-1v-3.5l5,4v-11L41,34.5z"
+          />
+        </svg>
       `;
-      svgNode.setAttribute('fill', color);
-      return svgNode;
+      return {
+        content,
+      };
     }
-
-    const customLogo = this.element_.createLoaderLogo();
-    return customLogo || null;
+    return this.element_.createLoaderLogo() || {};
```

Comment:
Let's make this return an empty object by default

Reply:
Done.